### DISCUSSION TITLE
/dev for catch-all routes

### DISCUSSION THREAD

**Discussion by ggomaeng** [wevm/frog-archived#159] [Q&A]
I can't open the devtool when using a regular expression `/:contractType/:chain/:symbol{.+}` . 

Am I doing something wrong? :c

![image](https://github.com/wevm/frog/assets/18402191/06c8a5a2-f9c6-4fff-8fdb-3c52969af960)

**Comment by tmm** [comment]
Can you share a minimal reproduction?

**Reply by ggomaeng** [reply]
here you go

https://github.com/ggomaeng/frog-catchall-route-bug
